### PR TITLE
React packages build system + react-code and react-form-validation packages

### PR DESCRIPTION
**Pull request by ganlhi**
This PR adds some tooling to develop, build and document React packages, and two actual React packages. 

## Tooling

By running `yarn build`, the React packages are built with Rollup. 
A Storybook instance can be built and pushed to a github page on this repository. 

## New packages

- `@heetch/react-code` provides a `Code` component to display highlighted code in the following languages:
  - javascript
  - jsx
  - typescript
  - tsx
  - yaml
  - json
- `@heetch/react-form-validation` provides a `FormWithValidation` wrapper component, built on top of `react-final-form`, which allows to define a custom validation rules set.

### CODE REVIEW

**Review by keanan — APPROVED** [review]
Great work!

File: packages/react-form-validation/__tests__/react-form-validation.test.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+describe('@heetch/react-form-validation', () => {
+  it('needs tests');
```
Comment (tamarasaurus) [inline packages/react-form-validation/__tests__/react-form-validation.test.js:2]: Do you plan on setting up the tests / storybook github pages deployment in a different PR?

**Review by tamarasaurus — APPROVED** [review]
:clap:

**Review by agathepey — APPROVED** [review]
👏🙏

File: packages/react-form-validation/__tests__/react-form-validation.test.js (hunk 1 above)
Comment (ganlhi) [reply packages/react-form-validation/__tests__/react-form-validation.test.js:2]: I still need to add tests indeed. Regarding the automated deployment of Storybook, I don't really know how to setup it within Github. It should be doable from a dev's computer though. I will test it.

**Pull request merged**